src/style.css

```
@@ -0,0 +1,111 @@
+/* #region general */
+@font-face {
+  font-family: "Avenir-Book";
+  src: local("Avenir-Book.ttf");
+  font-weight: 300;
+}
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: local("Avenir-Heavy.ttf");
+  font-weight: 300;
+}
+
+.visually-hidden {
+  position: absolute;
+  left: -9999px;
+}
+/* #endregion */
+
+/* #region search-bar */
+.search-bar {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  box-sizing: border-box;
+  height: fit-content;
+  width: 100%;
+  margin-top: 20px;
+  border-radius: 4px;
+}
```

unnecessary properties
```suggestion
.search-bar {
  position: relative;
  box-sizing: border-box;
  margin-top: 20px;
  border-radius: 4px;
}
```